python/tvm/topi/x86/depthwise_conv2d.py

```diff
@@ -305,6 +305,36 @@ def _schedule_depthwise_conv2d_NCHWc_impl(s, cfg, data_vec, kernel_vec, conv_out
     return s
 
 
+def schedule_depthwise_conv2d_nhwc(outs):
```

Comment:
@junrushao1994 @mbrookhart  is this location ok? i believe this schedule is generic. could you take a look at it?

Reply:
It looks like x86 always packs depthwise conv to NCHWc, so this will never get hit on an x86 machine. That makes it feel a little misplaced. Any reason not to put it here? https://github.com/apache/tvm/blob/main/python/tvm/topi/generic/conv2d.py
I think that's where a lot of the multi-CPU specialization ends up, i.e, all of the int8 kernels for ARM and X86 go through there.